.github/workflows/build-test.yml

```diff
@@ -453,7 +458,7 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         # keep args in sync with `clippy-all` in .cargo/config.toml
-        args: --all-targets --all-features -- -D warnings
+        args: --all-targets --all-features -- -D warnings -Aclippy::unknown-clippy-lints -Aclippy::field-reassign-with-default
```

Comment:
Issue: Please revert this change.  It may have originated from a messy merge if you pulled in one of my commits and then rebased it to a newer version.